ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -638,10 +640,12 @@ nnfw_invoke (const GstTensorFilterProperties * prop,
     void **private_data, const GstTensorMemory * input,
     GstTensorMemory * output)
 {
+  int64_t start_time, stop_time;
   nnfw_pdata *pdata;
   int err = 0;
   NNFW_STATUS nnfw_status;
 
+  start_time = g_get_real_time ();
```

Comment:
Discussion:
https://developer.gnome.org/glib/stable/glib-Date-and-Time-Functions.html#g-get-real-time
`g_get_monotonic_time() is probably more useful for measuring intervals.`
use g_get_monotonic_time()?

Reply:
`g_get_monotonic_time` is better. Updated.